### PR TITLE
Disable the modification of a shunt compensator with multiple sections

### PR DESCRIPTION
**Pull request by FranckLecuyer**
To be merged together with PR : https://github.com/gridsuite/gridstudy-app/pull/1410

### CODE REVIEW

File: src/main/java/org/gridsuite/modification/server/modifications/ShuntCompensatorModification.java [hunk 1]
```diff
@@ -16,14 +16,15 @@
...
+import org.gridsuite.modification.server.dto.OperationType;
 import org.gridsuite.modification.server.dto.ShuntCompensatorModificationInfos;
 import org.gridsuite.modification.server.dto.ShuntCompensatorType;
 
 import java.util.ArrayList;
 import java.util.List;
 
-import static org.gridsuite.modification.server.NetworkModificationException.Type.SHUNT_COMPENSATOR_NOT_FOUND;
-import static org.gridsuite.modification.server.NetworkModificationException.Type.VOLTAGE_LEVEL_NOT_FOUND;
+import static org.gridsuite.modification.server.NetworkModificationException.Type.*;
```
Comment (Walid-Sahnoun) [inline src/main/java/org/gridsuite/modification/server/modifications/ShuntCompensatorModification.java:27]: no need to import all types

File: src/main/java/org/gridsuite/modification/server/modifications/ShuntCompensatorModification.java [hunk 2]
```diff
@@ -62,11 +79,11 @@ public void apply(Network network, Reporter subReporter) {
         if (shuntCompensator.getModelType() == ShuntCompensatorModelType.LINEAR) {
             applyModificationOnLinearModel(subReporter, shuntCompensator, voltageLevel);
         }
-
     }
 
     private void applyModificationOnLinearModel(Reporter subReporter, ShuntCompensator shuntCompensator, VoltageLevel voltageLevel) {
         List<Report> reports = new ArrayList<>();
+
```
Comment (Walid-Sahnoun) [inline src/main/java/org/gridsuite/modification/server/modifications/ShuntCompensatorModification.java:86]: no need to this empty new line

File: src/main/java/org/gridsuite/modification/server/modifications/ShuntCompensatorModification.java (hunk 1 above)
Comment (FranckLecuyer) [reply src/main/java/org/gridsuite/modification/server/modifications/ShuntCompensatorModification.java]: Done

File: src/main/java/org/gridsuite/modification/server/modifications/ShuntCompensatorModification.java (hunk 2 above)
Comment (FranckLecuyer) [reply src/main/java/org/gridsuite/modification/server/modifications/ShuntCompensatorModification.java]: Removed

**Review by Walid-Sahnoun — APPROVED** [review]
code ok
test ok

**Review by YenguiSeddik — APPROVED** [review]

**Pull request merged**